onnx/defs/tensor_util.cc

```diff
@@ -54,5 +57,29 @@ DEFINE_PARSE_DATA(int64_t, int64s)
 DEFINE_PARSE_DATA(float, floats)
 DEFINE_PARSE_DATA(double, doubles)
 DEFINE_PARSE_DATA(uint64_t, uint64s)
+DEFINE_PARSE_DATA(uint8_t, int32s)
+DEFINE_PARSE_DATA(int8_t, int32s)
+DEFINE_PARSE_DATA(uint16_t, int32s)
+DEFINE_PARSE_DATA(int16_t, int32s)
+DEFINE_PARSE_DATA(uint32_t, uint64s)
```

Comment:
Now these ParseData for these new data types are not used in onnx repo. Will they all be covered in onnx optimizer?

Reply:
Good idea, what is your opinion ? @daquexian

Reply:
Maybe we can add these new template specifications in onnx-optimizer repo so that no unused functions are introduced in onnx repo.

Reply:
yeah, it is wise to implement function for these new data types in onnx-optimizer. I will undo this change in onnx repo. cc @daquexian @jcwchen